src/systems/systems/src/hydraulic/landing_gear.rs

```diff
@@ -577,11 +598,33 @@ impl GearSystemComponentHydraulicController {
         } else {
             Ratio::new::<ratio>(0.)
         };
+
+        self.update_jamming();
+    }
+
+    fn update_jamming(&mut self) {
+        // If jamming and actuator reaches jammed position, we activate the jamming
+        if self.jammed_actuator_failure.is_active()
+            && (self.jamming_position - self.actual_position)
+                .abs()
+                .get::<ratio>()
+                < 0.2
```

Comment:
**thought:** couldn't this be skipped with a high frametime ?

Reply:
Theoretically, it could be skipped in case of extreme gear speed motion which should not happen. It won't be impacted by frame rate as those physics are always solved at 100hz in sub steps